### PR TITLE
Update developer docs for mongoDB backups

### PR DESCRIPTION
**Pull request by rtrinque**
Update developer docs for mongoDB backups

### CODE REVIEW

File: source/manual/mongodb.html.md [hunk 1]
```diff
@@ -5,29 +5,13 @@ layout: manual_layout
...
-
-The on-site backup machine (`backup-1.management`) pulls the latest backup and stores it on disk. [Duplicity](http://duplicity.nongnu.org/) runs nightly and sends encrypted backups to an AWS S3 bucket.
-
-### Restoring
-
-- Fetch a backup from either the dedicated mount point, the on-site machine, or the S3 bucket [using Duplicity](restore-from-offsite-backups.html) (you may need a password kept in the encrypted [hieradata](https://github.com/alphagov/govuk-secrets)).
-- Unzip the file. This will produce a directory of data.
-- Run the command: `mongo restore --drop <directory>`
-
 ## mongodumps to S3
```
Comment (sengi) [inline source/manual/mongodb.html.md:12]: Is this bit dead now as well? Is there anything left on the old mongodump arrangements that wasn't moved to govuk-env-sync (as described below)?

**Review by sengi — APPROVED** [review]
Just one minor comment/question but doesn't need to block merging this (I should have ticked approve earlier, sorry).

File: source/manual/mongodb.html.md (hunk 1 above)
Comment (rtrinque) [reply source/manual/mongodb.html.md:12]: I was about to reply 'no it's all gone' and then I remembered performance platform uses mongodb *sob*

Comment (sengi) [reply source/manual/mongodb.html.md:12]: oh no 😭 no worries, thanks for updating all these!

**Pull request merged**